cli/Cargo.toml

```diff
@@ -64,7 +64,7 @@ experimental = [
 ]
 
 command = ["cylinder", "hex", "protobuf", "reqwest", "serde", "serde_json"]
-playlist = ["cylinder", "serde_json"]
+playlist = ["cylinder", "serde_json", "transact/family-command-workload", "transact/family-smallbank-workload"]
```

Comment:
Added a commit to rename this feature to `playlist-smallbank`, because playlist only supports smallbank as a workload type at the moment. In the future if other workload types are added to playlist, new features can be added with the name `playlist-<workload-type>`